app/views/content/index.html.erb

```diff
@@ -32,19 +32,21 @@
         <% end %>
       </ul>
 
-      <h2 id="get-support-and-guidance" class="govuk-heading-l eyfs-top-nav-anchors">Get support and guidance</h2>
-      <ul class="govuk-grid-row eyfs-card-group">
-          <li class="govuk-grid-column-one-half eyfs-card-group__item">
-            <div class="eyfs-card eyfs-card--clickable eyfs-card--link">
-              <div class="eyfs-card__content">
-                <a class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold govuk-link--no-visited-state" href="/articles">Support articles</a>
-                <svg class="eyfs-card--icon" width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
-                  <path d="M9 4.49992L20.0217 15.5213L9 26.5427" stroke="#1D70B8" stroke-width="2"/>
-                </svg>
+      <%- if Feature.enabled?(:support_articles) %>
```

Comment:
Extremely minor: `<%-` is the same as `<%` in the current version of erb, so the `-` is redundant.